client/src/components/categorical/index.js

```diff
@@ -185,7 +185,8 @@ class Categories extends React.Component {
         {/* READ ONLY CATEGORICAL FIELDS */}
         {/* this is duplicative but flat, could be abstracted */}
         {allCategoryNames.map((catName) =>
-          !schema.annotations.obsByName[catName].writable ? (
+          !schema.annotations.obsByName[catName].writable &&
+          schema.annotations.obsByName[catName].categories.length > 1 ? (
```

Comment:
Categories might not be defined for the category

```suggestion
          !schema.annotations.obsByName[catName].writable &&
          schema.annotations.obsByName[catName].categories?.length > 1 ? (
```

Reply:
Actually my fix is not correct, we want categories with undefined `categories` values to pass through to render:

```suggestion
          !schema.annotations.obsByName[catName].writable &&
          (schema.annotations.obsByName[catName].categories?.length > 1 ||
            !schema.annotations.obsByName[catName].categories) ? (
```